### ISSUE TITLE
Too many logs in the driver initialization

### ISSUE THREAD

**Issue by hongbolv** [intel/libva#538]
When every process initializes the media driver, there are always fine lines log in the terminal. In the server side, if we have multiple channels, the log size will increase to n*5 lines. Then it is not easy to find the useful log.

**Comment by XinfengZhang** [comment]
@hongbolv , what you mentioned is these log?
libva info: VA-API version 1.13.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /home/intel/carl/open_driver/build/media_driver/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_13
libva info: va_openDriver() returns 0

**Comment by pldin601** [comment]
@XinfengZhang I see the same logs during my gstreamer app initialization. Is there some env var should I use to disable this output?

**Comment by XinfengZhang** [comment]
yes, you could use LIBVA_MESSAGING_LEVEL  to set the level, also , there are one discussion about default message level https://github.com/intel/libva/pull/564
